.gitignore

```diff
@@ -15,3 +15,6 @@ bin/
 
 /text-ui-test/ACTUAL.txt
 text-ui-test/EXPECTED-UNIX.TXT
+
+SaveData.txt
+/src/main/resources/images/*.png
```

Comment:
Hi! I cloned your project, but was unable to run it because these images have been gitignore'd,
meaning the application throws `NullPointerExceptions` as the images can't be found and loaded.